src/chrome/content/rules/ComputerWoche.de.xml

```diff
@@ -0,0 +1,6 @@
+<ruleset name="ComputerWoche">
+    <target host="*.computerwoche.de" />
+
+    <rule from="^http://(images|www)\.computerwoche\.de/"
+        to="https://$1.computerwoche.de/" />
```

Comment:
Please push your "I removed it." change into https://github.com/numismatika/https-everywhere/tree/rules_19.12.14